libs/platform/src/lib/components/form/multi-input/multi-input.component.scss

```diff
@@ -0,0 +1,32 @@
+.popover-mulit-input-custom {
```

Comment:
Is it temporary?

Reply:
nope

Reply:
nope, necessary to adapt the width to the parent component.